articles/monitoring/guides/monitor-applications.md

```diff
@@ -13,20 +13,48 @@ useCase:
   - synthetic-transactions
 ---
 # Monitor Applications
+Auth0 is a critical dependency of your application and/or service. Monitoring Auth0's health can be important as it allows you to report specific errors to your customers and/or take mitigating actions in case of an issue with Auth0.
 
-If you would like to monitor your own [application](/applications) or conduct end-to-end testing, you’ll need to set up your own tests. 
+You can monitor Auth0 using many approaches, each of them complementary with the others. Picking the approaches to use depends on your needs and investment possibilities.
 
-If you've extended Auth0 through [rules](/rules) or a [custom database connection](/connections/database/custom-db), you can build a synthetic transaction that exercises these capabilities using the [Resource Owner Password Grant](/api-auth/tutorials/password-grant). One way of doing this is to [Monitor Auth0 Using SCOM](/monitoring/guides/monitor-using-SCOM).
+## Synthetic transactions
+The simplest approach to monitor Auth0. This involves setting up a periodic request to perform an authentication transaction. If the request works successfully, Auth0 is working fine. If the request fails this _might_ indicate an issue with Auth0.
```

Comment:
The simplest approach to monitor Auth0:

- Set up a periodic request to perform an authentication transaction.
- If the request succeeds, Auth0 is working fine.
- If the request fails, this may indicate:
  - an issue with Auth0
  - an issue specific to the tenant used for the synthetic transaction
  - or just a single failed request.